decentraland/components/presentational/LinkToEtherscan.js

```diff
@@ -1,34 +1,44 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Button, Icon } from "native-base";
-import { showError, openURL } from "@helpers";
+import {
+  showError,
+  openURL,
+  getNetworkName,
+  getTransactionHashFromAction,
+} from "@helpers";
 
-const _openEtherscanOf = async address => {
-  const url = `https://etherscan.io/address/${address}`;
+const supportedNetworks = ["ropsten"];
+
+const _openEtherscanOf = async action => {
+  const networkName = getNetworkName();
+  const transactionHash = await getTransactionHashFromAction(action);
```

Comment:
Nice, this is looking great

Reply:
Maybe we call this variable `actionId` to keep it meaningful to people who aren't Etherum-savvy?

Reply:
I've created a function `buildEtherscanUrlFromAction` to get the URL, inside of it I've kept the `transactionHash` var name to use the same language as `etherscan` does, make sense?
Refs: https://github.com/tasitlabs/tasit/pull/260#discussion_r272744128

Reply:
That function is a good addition. Okay, we can keep that var name the same. I'll keep pondering how strongly I feel about the `actionId` idea. It's only a weakly held opinion for now. If we do move away from Etherscan, using the same variable name as them becomes less important (although most block explorers are likely to use a similar name).

Resolving this.